app/src/main/java/com/example/socialmediavbsanalay/presentation/fragments/MessageFragment.kt

```diff
@@ -0,0 +1,90 @@
+package com.example.socialmediavbsanalay.presentation.fragments
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.navigation.Navigation
+import androidx.recyclerview.widget.LinearLayoutManager
+import com.example.socialmediavbsanalay.R
+import com.example.socialmediavbsanalay.databinding.FragmentMainPageBinding
+import com.example.socialmediavbsanalay.databinding.FragmentMessageBinding
+import com.example.socialmediavbsanalay.presentation.adapters.MessageItemAdaptor
+import com.example.socialmediavbsanalay.presentation.adapters.StoryAdapter
+import dagger.hilt.android.AndroidEntryPoint
+
+
+
+@AndroidEntryPoint
+class MessageFragment : Fragment() {
+
+    private var _binding: FragmentMessageBinding? = null
+    private val binding get() = _binding!!
+    private lateinit var storyAdapter: StoryAdapter
+    private lateinit var messageItemAdaptor: MessageItemAdaptor
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        _binding = FragmentMessageBinding.inflate(inflater, container, false)
+        return binding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+
+
+        storyAdapter = StoryAdapter()
```

Comment:
adapterların oluşturulduğu kodları ve clickListenerların setlediği kodları ayrı bir fonksiyon oluşturup ( örneğin initRecyclerViews() ve initClickListeners() ) o fonksiyonların içine taşır mısın? Kod olarak bir yanlışlık yok ama daha clean gözükmesi açısından bu şekilde kullanılabilir.

Reply:
MainActivityDe çok iyi yaptığımı düşünüyorum onu :)